nemo_text_processing/inverse_text_normalization/es/taggers/cardinal.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+# Copyright 2015 and onwards Google, Inc.
```

Comment:
we don't need to add Google in header only in graphutils.py Could you delete all Google references?
We will also delete for English